projects/1.20/assets/icarus/icarus/lang/zh_cn.json

```
@@ -0,0 +1,113 @@
+  "item.icarus.lime_mechanical_feathered_wings": "青柠色机械羽翼",
+  "item.icarus.pink_mechanical_feathered_wings": "粉红色机械羽翼",
+  "item.icarus.gray_mechanical_feathered_wings": "灰色机械羽翼",
+  "item.icarus.light_gray_mechanical_feathered_wings": "淡灰色机械羽翼",
+  "item.icarus.cyan_mechanical_feathered_wings": "青色机械羽翼",
+  "item.icarus.purple_mechanical_feathered_wings": "紫色机械羽翼",
+  "item.icarus.blue_mechanical_feathered_wings": "蓝色机械羽翼",
+  "item.icarus.brown_mechanical_feathered_wings": "棕色机械羽翼",
+  "item.icarus.green_mechanical_feathered_wings": "绿色机械羽翼",
+  "item.icarus.red_mechanical_feathered_wings": "红色机械羽翼",
+  "item.icarus.black_mechanical_feathered_wings": "黑色机械羽翼",
+
+  "item.icarus.white_mechanical_leather_wings": "白色机械皮翼",
+  "item.icarus.orange_mechanical_leather_wings": "橙色机械皮翼",
+  "item.icarus.magenta_mechanical_leather_wings": "洋红色机械皮翼",
+  "item.icarus.light_blue_mechanical_leather_wings": "淡蓝色机械皮翼",
+  "item.icarus.yellow_mechanical_leather_wings": "黄色机械皮翼",
+  "item.icarus.lime_mechanical_leather_wings": "青柠色机械皮翼",
+  "item.icarus.pink_mechanical_leather_wings": "粉红色机械皮翼",
+  "item.icarus.gray_mechanical_leather_wings": "灰色机械皮翼",
+  "item.icarus.light_gray_mechanical_leather_wings": "淡灰色机械皮翼",
+  "item.icarus.cyan_mechanical_leather_wings": "青色机械皮翼",
+  "item.icarus.purple_mechanical_leather_wings": "紫色机械皮翼",
+  "item.icarus.blue_mechanical_leather_wings": "蓝色机械皮翼",
+  "item.icarus.brown_mechanical_leather_wings": "棕色机械皮翼",
+  "item.icarus.green_mechanical_leather_wings": "绿色机械皮翼",
+  "item.icarus.red_mechanical_leather_wings": "红色机械皮翼",
+  "item.icarus.black_mechanical_leather_wings": "黑色机械皮翼",
+
+  "item.icarus.white_light_wings": "白色光翼",
+  "item.icarus.orange_light_wings": "橙色光翼",
+  "item.icarus.magenta_light_wings": "洋红色光翼",
+  "item.icarus.light_blue_light_wings": "淡蓝色光翼",
+  "item.icarus.yellow_light_wings": "黄色光翼",
+  "item.icarus.lime_light_wings": "青柠色光翼",
+  "item.icarus.pink_light_wings": "粉红色光翼",
+  "item.icarus.gray_light_wings": "灰色光翼",
+  "item.icarus.light_gray_light_wings": "淡灰色光翼",
+  "item.icarus.cyan_light_wings": "青色光翼",
+  "item.icarus.purple_light_wings": "紫色光翼",
+  "item.icarus.blue_light_wings": "蓝色光翼",
+  "item.icarus.brown_light_wings": "棕色光翼",
+  "item.icarus.green_light_wings": "绿色光翼",
+  "item.icarus.red_light_wings": "红色光翼",
+  "item.icarus.black_light_wings": "黑色光翼",
+
+  "item.icarus.flandres_wings": "芙兰朵露之翼",
+  "item.icarus.discords_wings": "失谐之翼",
+  "item.icarus.zanzas_wings": "赞扎之翼",
+
+  "config.icarus.can_loop_de_loop_server": "（服务器）玩家是否可以翻筋斗",
+  "config.icarus.can_loop_de_loop_client": "（客户端）玩家是否可以翻筋斗",
+  "config.icarus.armor_slows": "盔甲是否会减慢玩家的速度",
+  "config.icarus.can_slow_fall": "玩家是否可以在飞行时通过潜行来实现缓慢降落",
+  "config.icarus.max_slowed_multiplier": "穿戴盔甲的最大减速倍数",
+  "config.icarus.wings_speed": "翅膀飞行速度",
+  "config.icarus.wings_durability": "所有翅膀的耐久度",
+  "config.icarus.exhaustion_amount": "飞行消耗的饥饿值",
+  "config.icarus.required_food_amount": "飞行需要多少饥饿值",
```

per [mcw:饥饿#机制](https://zh.minecraft.wiki/w/%E9%A5%A5%E9%A5%BF#%E6%9C%BA%E5%88%B6)

```suggestion
  "config.icarus.exhaustion_amount": "飞行的饥饿消耗度",
  "config.icarus.required_food_amount": "飞行所需的饥饿值下限",
```